source/.res/text/important.percona-release.latest.yum.txt

```diff
@@ -0,0 +1,9 @@
+
+.. important::
+
+   If |percona-release| is already installed on the target system, make sure
+   you are using the latest version of |percona-release|:
+
+   .. code-block:: bash
+
+      $ sudo yum update percona-release
```

Comment:
this command will work only if original repo is enabled

Reply:
So the command must be preceded with the command:
$ sudo percona-release enable original 
Correct?

I believe the same is valid for DEB-based systems

Reply:
@nastena1606 this is more complicated question. for example the might be package conflict while having orignal repo and some other repos.
another solution is to download latest percona-release package from web and install it.
for rpms it can be yum install <url-to_latest.rpm>
for debs - download and install

Reply:
But this is the thing. The warning states that if the user has percona-release already installed, there's the command to check for the latest version. Unfortunately there's no command (documented at least) to check what percona-release is installed / enabled (https://www.percona.com/doc/percona-repo-config/percona-release.html#commands)

If the solution is to download the latest percona-release package and this resolves package conflicts and/or disables unrequired repos to ensure the correct one is used, then there's no need in this warning at all.